core/src/com/unciv/models/ruleset/unit/BaseUnit.kt

```diff
@@ -30,6 +30,8 @@ class BaseUnit : INamed, IConstruction {
     lateinit var unitType: UnitType
     var requiredTech:String? = null
     var requiredResource:String? = null
+    var requiredWonder:String? = null
+    var requiredPolicy:String? = null
```

Comment:
Same as with Required Wonder, this should really be a unique.

Reply:
redone